clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -414,8 +435,8 @@ def get_logs(self, logging_dir):
                 log_file_name = "{pod}_{num}.log".format(
                     pod=pod.metadata.name, num=str(i))
                 log_file_alt = "{cname}.log".format(cname=c.name)
-                logger.info("log file name: {}".format(log_file_name))
-                logger.info("log alt file name: {}".format(log_file_alt))
+                self.logger.info("log file name: {}".format(log_file_name))
+                self.logger.info("log alt file name: {}".format(log_file_alt))
```

Comment:
What is the purpose of `log_file_alt`? It doesn't look like it's being used at all.

Reply:
frankly i have no idea. I just deleted this line